### PR TITLE
Implementing Calendar View in the user portal of talawa

### LINKED ISSUE #1202: Implementing Calendar View in the user portal of talawa
**Describe the bug**
In the user portal which can be accessed here `http://localhost:3000/user` needs to be implemented with a calendar view in the event screen.

**To Reproduce**

1. run the talawa admin
2. access the user login form and log in as a user
3. click on event tab on navbar
4. access the issues

**Expected behavior**
A calendar view similar to talawa-admin

**Actual behavior**
No calendar view

https://github.com/PalisadoesFoundation/talawa-admin/assets/73706697/2a66ef14-a83e-4273-a5c8-931c45c46a32

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by chandel-aman**
**What kind of change does this PR introduce?**

Feature

**Issue Number:**

Fixes #1202 

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

image

**If relevant, did you update the documentation?**

NA

**Summary**

Added calendar view of events in user portal of talawa for better user experience.

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by chandel-aman** [comment]
@Cioppolo14 @aashimawadhwa 
Could you please review this PR?

File: src/screens/UserPortal/Events/Events.tsx [hunk 1]
```diff
@@ -76,8 +80,15 @@ export default function events(): JSX.Element {
...
   });
 
+  const { data: orgData } = useQuery(ORGANIZATIONS_LIST, {
+    variables: { id: organizationId },
+  });
+
   const [create] = useMutation(CREATE_EVENT_MUTATION);
 
+  const userId = localStorage.getItem('id') as string;
+  const userRole = localStorage.getItem('UserType') as string;
```
Comment (beingnoble03) [inline src/screens/UserPortal/Events/Events.tsx:90]: I'm not sure if `UserType` sets in the local storage, when we login directly from the User Portal. Please check. I think in that case, we'll face error as you have used the `as` keyword here.

**Review by beingnoble03 — COMMENTED** [review]
Please address the above comment.

File: src/screens/UserPortal/Events/Events.tsx (hunk 1 above)
Comment (beingnoble03) [reply src/screens/UserPortal/Events/Events.tsx:90]: Also check for `id`. Please fix it if it could be `undefined`.

Comment (chandel-aman) [reply src/screens/UserPortal/Events/Events.tsx:90]: image


image

Yes, we are storing `UserType` and `id` in local storage both in user as well as admin portal.

**Comment by chandel-aman** [comment]
@beingnoble03
Yes, we are storing `UserType` and `id` in local storage of browser as could you see in the screenshot and code provided.
If there is nothing else that needs to be reviewed could you please approve this PR?

**Comment by beingnoble03** [comment]
@chandel-aman did you use User Portal to login?

**Comment by chandel-aman** [comment]
> @chandel-aman did you use User Portal to login?

Yes, I did login from user portal.

**Review by beingnoble03 — APPROVED** [review]

**Comment by palisadoes** [comment]
@rishav-jha-mech @aashimawadhwa PTAL

**Review by aashimawadhwa — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**